tests/comfy_cli/command/models/test_models.py

```
@@ -304,3 +304,109 @@ def test_huggingface_url_with_folder_structure():
         None,
         "main",
     )
+
+
+# ---------------------------------------------------------------------------
+# --downloader CLI option tests
+# ---------------------------------------------------------------------------
+
+
+class TestDownloadCommandDownloaderOption:
```

Pylint (4.0.5)

[convention] Missing class docstring

(C0115)